src/style.css

```diff
@@ -0,0 +1,41 @@
+* {
+  margin: 0;
```

Comment:
reset margins only for specific tags not for all
`*` selector better to use only for box-sizing